mystery-deckbuilder/Assets/Scripts/NPC/Nibbles/NibblesDialogueTrigger.cs

```diff
@@ -0,0 +1,17 @@
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+/* A placeholder dialogue trigger class for the alpha */
+public class NibblesDialogueTrigger : MonoBehaviour
+{
+
+    //will be called when player clicks Nibbles
+    public void StartDialogue1()
+    {
+        Debug.Log("triggered dialogue");
+        DialogueTree tree = transform.GetComponent<NibblesIntro>().dialogueTree;
+        string name = transform.GetComponent<NPC>().name;
+        DialogueBoxManager.Instance.StartDialogue(tree, name);
+    }
```

Comment:
At some point in the future, you could keep the current Dialogue instance reference as a GameStateValue. That way, other UIs (say the map) could close themselves if a dialogue instance is generated